Homework_4.py

```diff
@@ -0,0 +1,15 @@
+#1. Створити програму, яка буде очікувати введення тексту від користувача і повідомляти —
+#чи є введений текст “числом” чи “словом”.
+#2. Якщо введений текст “число”, програма має також вказати чи є воно парним чи непарним.
+#3. Якщо це “слово”, програма має вказати його довжину.
+
+def type_checker():
+    enter = input("Enter your text ")
+    if(enter.isdigit() == True):
+        if (float(enter) % 2 == 0):
```

Comment:
It is better to use here int() transformation since according to .isdigit() check the `enter` value cannot contain a dot